sources/api/bootstrap-commands/src/main.rs

```diff
@@ -0,0 +1,315 @@
+/*
```

Comment:
This needs to be a doc comment, or it won't populate the template used to render `README.md`
```suggestion
/*!
```